competitions/100Change2020/ansible/100Change2020.yml

```diff
@@ -12,11 +12,11 @@
   roles:
     - mediawiki
     - activitylog
-    - collection
+    - simplebook
     - embed_video
     - permissions
     - simplefavorites
-    # PickSome has to come after collection so the left menu bar is correct
+    # PickSome has to come after simplebook so the left menu bar is correct
```

Comment:
Okay, the original comment was odd too, in that it proper-noun'd "PickSome" but not "Collection".  I'd say let's follow the convention "PickSome" started and write "SimpleBook" here.

(Same in repeat occurrences below, of course.)

Reply:
Fixed!